rules/windows/process_creation/proc_creation_win_susp_image_missing.yml

```diff
@@ -23,10 +23,12 @@ detection:
             - ''
     filter_4688:
         - Image:
+            - 'System'
             - 'Registry'
             - 'MemCompression'
             - 'vmmem'
         - CommandLine:
+            - 'System'
```

Comment:
Can you link to the FP where System was in the Command line? Just to put it as a future reference?